akka-actor/src/main/resources/reference.conf

```diff
@@ -906,5 +906,59 @@ akka {
     }
   }
 
+  coordinated-shutdown {
+    # The timeout that will be used for a phase if not specified with 
+    # 'timeout' in the phase
+    default-timeout = 5 s
```

Comment:
`default-phase-timeout`? In order not to be confused with "the entire shutdown timeout"?